components/IssueCredentialOptions.yml

```diff
@@ -38,6 +38,17 @@ components:
             auto-generate a credentialId if one is not supplied by the issuer
             coordinator, because doing so could create a partitioning error if the
             result is never received by the client.
+        proofId:
+          type: string
+          description: >
+            An optional identifier for the proof being added by this issuer instance. 
+            If not provided, the issuer service MAY auto-generate a proof identifier 
+            based on its configuration.
+        previousProof:
+          type: object
+          description: >
+            Optional reference to previous proof(s) when creating proof chains. 
+            The configuration of the issuer instance determines how this is processed.
```

Comment:
Hmm, interested @dlongley's take on this as an implementer.

This should probably be a `string` that refers to a previous `proofId`. Specifying `proofId` feels useful too, but it should probably be a globally unique identifier with a scheme prefix, such as `urn:uuid:d6764dfc-6b33-11f0-8e93-63e78a307c56` or `https://domain.example/proofs#82978473729973`.

Reply:
See above: https://github.com/w3c-ccg/vc-api/pull/499/files#r2240519905